packages/office-ui-fabric-react/src/utilities/color/rgb2hex.ts

```diff
@@ -0,0 +1,15 @@
+import { MAX_COLOR_RGB } from './consts';
+import { clamp } from './clamp';
+
+/** Converts RGB components to a hex color string (without # prefix). */
+export function rgb2hex(r: number, g: number, b: number): string {
+  return [_rgbToPaddedHex(r), _rgbToPaddedHex(g), _rgbToPaddedHex(b)].join('');
+}
+
+/** Converts an RGB component to a 0-padded hex component of length 2. */
```

Comment:
What does a 0 padded hex mean? I can get that it's `0<somehex>` but why is it used?

Reply:
Pretty sure "padded" in this case just means "a valid value" and not a negative number or something greater than FF.

All this code is copied and pasted, all I did is move it. Improvements can/should be made in a subsequent PR.